src/utils/envUtils.ts

```diff
@@ -0,0 +1,63 @@
+import env from "../env";
+import { Logger } from "./logger";
+
+/**
+ * A regular expression to match the release version from the build tag.
+ *
+ * @see {@link parseReleaseVersion} for a usage example.
+ */
+export const ReleaseRegex = /^(\d{1,4}\.\d{1,4}\.\d{1,4}).*/;
+
+/**
+ * Safely parse the release version from the environment variable.
+ * If it is not set, return an empty string.
+ *
+ * @note This utility expects the current tag to be defined as `X.X.X.buildNumber`.
+ * @see {@link AppEnv.REACT_APP_FE_VERSION}
+ * @returns The parsed release version or "N/A" if not set.
+ */
+export const parseReleaseVersion = (): string => {
+  const { REACT_APP_FE_VERSION } = env || {};
+
+  if (!REACT_APP_FE_VERSION || typeof REACT_APP_FE_VERSION !== "string") {
+    Logger.error("parseReleaseVersion: REACT_APP_FE_VERSION is not set or is not a string");
+    return "N/A";
+  }
+
+  if (!ReleaseRegex.test(REACT_APP_FE_VERSION)) {
+    Logger.error(
+      `parseReleaseVersion: REACT_APP_FE_VERSION is not in the expected format: ${REACT_APP_FE_VERSION}`
+    );
+    return "N/A";
+  }
+
+  const splitVersion = REACT_APP_FE_VERSION.match(ReleaseRegex);
+  if (!splitVersion || splitVersion.length < 2 || !splitVersion[1]) {
+    Logger.error(
+      `parseReleaseVersion: Unable to get release version from build tag: ${REACT_APP_FE_VERSION}`
+    );
+    return "N/A";
+  }
+
+  return splitVersion[1];
```

Comment:
More of a question, the correct version number is displaying based on the user story, but maybe it is worth double checking if the build number should be included in the version number? All other bento-based projects show the build number, so I just wonder if this is intentional or not.

Disregard, if the way you have it now is how they want to handle this.

Reply:
I wrote the original story based on the Bento footers with the full FE/BE build tags included, but KC said it would be too technical to include it. FWIW, I do agree with you though.